turbine/src/quic_endpoint.rs

```diff
@@ -39,10 +40,17 @@ use {
 const CLIENT_CHANNEL_BUFFER: usize = 1 << 14;
 const ROUTER_CHANNEL_BUFFER: usize = 64;
 const CONNECTION_CACHE_CAPACITY: usize = 3072;
-const INITIAL_MAXIMUM_TRANSMISSION_UNIT: u16 = 1280;
 const ALPN_TURBINE_PROTOCOL_ID: &[u8] = b"solana-turbine";
 const CONNECT_SERVER_NAME: &str = "solana-turbine";
 
+// Transport config.
+const DATAGRAM_RECEIVE_BUFFER_SIZE: usize = 256 * 1024 * 1024;
+const DATAGRAM_SEND_BUFFER_SIZE: usize = 128 * 1024 * 1024;
+const INITIAL_MAXIMUM_TRANSMISSION_UNIT: u16 = MINIMUM_MAXIMUM_TRANSMISSION_UNIT;
+const KEEP_ALIVE_INTERVAL: Duration = Duration::from_secs(4);
+const MAX_IDLE_TIMEOUT: Duration = Duration::from_secs(10);
+const MINIMUM_MAXIMUM_TRANSMISSION_UNIT: u16 = 1280;
```

Comment:
Were these values found/picked empirically? The MTU and durations are seemingly simple; however, the buffer size ones seem a little less straight forward to me in how the numbers were picked

Reply:
For the buffer sizes I dug the default values in the quinn library and increased them by a couple of orders of magnitude because the peak of the turbine traffic can be pretty high.
https://docs.rs/quinn-proto/0.10.2/src/quinn_proto/config.rs.html#296-330

My primary goal here is to make these core configs explicit. Hopefully, the incremental testing on testnet will allow to further fine tune these if needed.

Reply:
Got it. Maybe if/once we determine that QUIC is a suitable replacement for turbine, we can circle back and leave some paper trail comments. Hypothetically, given a block duration and block size (in shreds), we should be able to come up with a ballpark "expected load".

Not saying I think these numbers have to be a "tight fit"; rather, some rough numbers + a comment of leaving sufficient margin would be nice for the next person who is looking through the code